src/modules/Builder/features/ActivityItems/LeftBar/Item/Item.tsx

```diff
@@ -27,10 +26,10 @@ export const Item = ({ item, name, activeItemId, onSetActiveItem, onRemoveItem }
 
   return (
     <StyledItem
-      isActive={activeItemId === item.id}
+      isActive={activeItemId === (item.id ?? item.key)}
```

Comment:
let's move this expression into a function since it was used in many places

Reply:
done